lib/rust/lexer/definition/src/lexer.rs

```diff
@@ -0,0 +1,1196 @@
+//! This module contains the definition of the lexer for the Enso programming language.
+
+use crate::prelude::*;
+use flexer::*;
+
+use crate::library::token::BlockType;
+use crate::library::token::Token;
+use crate::library::token;
+use flexer::automata::pattern::Pattern;
```

Comment:
Crate and non-crate use declarations should be separated with an empty line

```suggestion
use crate::library::token;

use flexer::automata::pattern::Pattern;
```